templates/pdb.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "zur.fullname" . }}
+spec:
+  maxUnavailable: {{ .Values.pdb.maxUnavailable }}
```

Comment:
let's change it to minAvailable rather than maxUnavailable, it's easier to work with in general in order to guarantee that you have enough pods to handle your workload